etna/transforms/outliers.py

```diff
@@ -30,6 +30,27 @@ def __init__(self, in_column: str):
         self.in_column = in_column
         self.outliers_timestamps: Optional[Dict[str, List[pd.Timestamp]]] = None
 
+    def _save_original_values(self, ts: TSDataset) -> "OutliersTransform":
+        """
+        Save values to be replaced with NaNs.
+
+        Parameters
+        ----------
+        ts:
+            original TSDataset
+
+        Returns
+        -------
+        instance with saved values
+        """
+        self.original_values = dict()
```

Comment:
Add `self.original_values: Optional[Dict[str, List[pd.Timestamp]]] = None` to the constructor